packages/api-reference/src/components/Content/Lazy/Lazy.vue

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
+import { type EventBusKey, useEventBus } from '@vueuse/core'
+
+/**
+ * Setup an event bus so we can listen for loaded events
+ */
+const lazyEventBusKey: EventBusKey<{ id: string }> = Symbol('lazyEventBusKey')
+export const lazyBus = useEventBus(lazyEventBusKey)
```

Comment:
Lets drop this into a vanilla ts file. In general I find exporting/importing from .vue files generally hard to follow

Reply:
cool beans, just saw it elsewhere. Will do 🖖🏾